src/main/java/frc/robot/subsystems/Manipulator.java

```diff
@@ -0,0 +1,126 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import PIDControl.PIDControl;
+import PIDControl.PIDControl.Coefficient;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class Manipulator extends SubsystemBase 
+{
+    private static Manipulator _instance;
+
+    public static Manipulator getInstance()
+    {
+        if(_instance == null)
+        {
+            _instance = new Manipulator();
+        }
+
+        return _instance;
+    }
+
+    //Wrist Controls
+    private CANSparkMax      _wristMotor;
+    private DutyCycleEncoder _wristEncoder;
+    private PIDControl       _wristPID;
+    //Twist Motion Controls
+    private CANSparkMax      _twistMotor;
+    private DutyCycleEncoder _twistEncoder;
+    private PIDControl       _twistPID;
+    //Hand Controls
+    private CANSparkMax      _graspMotor;
+    private DutyCycleEncoder _graspEncoder;
+    private PIDControl       _graspPID;
+    //Intake Controls
+    private CANSparkMax      _intakeMotor;
+    private DigitalInput     _intakeSensor;
```

Comment:
Add some whitespace between the groups to make this a little more readable:

```suggestion
    //Wrist Controls
    private CANSparkMax      _wristMotor;
    private DutyCycleEncoder _wristEncoder;
    private PIDControl       _wristPID;

    //Twist Motion Controls
    private CANSparkMax      _twistMotor;
    private DutyCycleEncoder _twistEncoder;
    private PIDControl       _twistPID;

    //Hand Controls
    private CANSparkMax      _graspMotor;
    private DutyCycleEncoder _graspEncoder;
    private PIDControl       _graspPID;

    //Intake Controls
    private CANSparkMax      _intakeMotor;
    private DigitalInput     _intakeSensor;
```